website/content/en/docs/advanced-topics/scorecard/custom-tests.md

```diff
@@ -295,9 +295,9 @@ connection to invoke the Kube API.
 [basic_tests]: https://github.com/operator-framework/operator-sdk/blob/master/internal/scorecard/tests/basic.go
 [config_yaml]: https://github.com/operator-framework/operator-sdk/blob/master/internal/scorecard/testdata/bundle/tests/scorecard/config.yaml
 [scorecard_main_func]: https://github.com/operator-framework/operator-sdk/blob/master/images/scorecard-test/cmd/test/main.go
-[custom_scorecard_repo]: https://github.com/operator-framework/operator-sdk/tree/master/internal/scorecard/examples
+[custom_scorecard_repo]: https://github.com/operator-framework/operator-sdk
 [user_doc]: /docs/advanced-topics/scorecard/scorecard/
-[scorecard_binary]: https://github.com/operator-framework/operator-sdk/blob/master/internal/scorecard/examples/custom-scorecard-tests/images/custom-scorecard-tests/cmd/test/main.go
-[sample_makefile]: https://github.com/operator-framework/operator-sdk/blob/master/internal/scorecard/examples/custom-scorecard-tests/Makefile
+[scorecard_binary]: https://github.com/operator-framework/operator-sdk/blob/master/internal/scorecard
+[sample_makefile]: https://github.com/operator-framework/operator-sdk/blob/master/internal/scorecard
```

Comment:
Both of these links point to the same code, and neither are really pointing to what they describe. Can you update/remove the sections these links are used in?

Reply:
I was going to update these in a docs PR after this one that fixes those links to the newly relocated locations, this is essentially just making the link-checker happy till I can get the docs PR merged next.

Reply:
it is because you are using the link. I think it should work point for the code directly. e.g operator-sdk/internal/scorecard/examples/custom-scorecard-tests/Makefile